### PR TITLE
Use new collection type names

### LINKED ISSUE #116: Refactor Strapi collection types

### PR DESCRIPTION
**Pull request by dhmacs**
This PR closes #116 

I'm still keeping the name "collections" in many parts of the codebase mostly to avoid conflicts with other branches that are ready to be reviewed after this PR. When those branches will be merged too then I'll refactor all references to "collection"

## QA

1. Visit the Vercel PR preview
2. Visit the parts collection page
3. Verify that product list page works with the new Strapi instance

### CODE REVIEW

File: lib/api/strapi/operations/getCollection.graphql [hunk 1]
```diff
@@ -24,6 +32,14 @@ query getCollectionPageData($whereCollection: JSON, $whereStoreSettings: JSON) {
                parent {
                   title
                   handle
+                  parent {
```
Comment (evannoronha) [inline lib/api/strapi/operations/getCollection.graphql:35]: Why are we looking farther up the hierarchy now than we were previously?

Comment (evannoronha) [inline lib/api/strapi/operations/getCollection.graphql:35]: Why are we looking farther up the hierarchy now than we were previously?

**Review by evannoronha — APPROVED** [review]
CR 🔥 

LGTM

**Comment by erinemay** [comment]
QA 🦑 The parts page "works" as far as:
- It gets loaded successfully
- There is product information populated
- Search and filtering works as expected


I'm noticing several things, and I want to make sure these are outside the scope of this PR.
Deploy_block 🔱 on confirming that the following is current expected behavior.

<details><summary>There's no clear organization to the model buttons.</summary>

I would expect some kind of ordering or rationale why specific models are displayed at the top.

image

</details>

<details><summary>Most of the parts collections linked in the model section are empty</summary>

image

</details>


<details><summary>Price displayed here is not the same as the price displayed on the product page.</summary>

The first part listed on the page is the `MacBook Pro 13" (A2251, 2020) 2.3 GHz Logic Board with Paired Touch ID Sensor`
When I open the link to the actual product page on ifixit.com, it appears that the current available product is priced differently.

image

image


</details>

File: lib/api/strapi/operations/getCollection.graphql (hunk 1 above)
Comment (dhmacs) [reply lib/api/strapi/operations/getCollection.graphql:35]: @evannoronha I was extending this to allow for more nesting and see if the query breaks (spoiler alert: if you add even more it does). During today meeting we decided to just show 2 or 3 ancestors in the breadcrumb, so eventually I'll simplify this

**Comment by GrantMatejka** [comment]
CR 🌽

**Comment by erinemay** [comment]
Update from @dhmacs. This PR is addressing the page as a whole loading and working. It does. 

un_deploy_block 👍 

Collection/model behavior noted above is currently expected.

**Comment by sterlinghirsh** [comment]
CR :zap:

**Comment by dhmacs** [comment]
> There's no clear organization to the model buttons.
> I would expect some kind of ordering or rationale why specific models are displayed at the top.
> 
> image

@erinemay Yeah you're right, the most straightforward thing to do is to sort alphabetically, but that might not be the best UX. Another option is to have the most recent devices displayed first. This would required to add something like a `releasedAt` field in the product list collection type. This is something that needs to be defined

**Pull request merged**